benchmarks/dynamo/pr_time_benchmarks/benchmark_base.py

```diff
@@ -65,6 +67,22 @@ class BenchmarkBase(ABC):
     # number of iterations used to run when collecting instruction_count or compile_time_instruction_count.
     _num_iterations = 5
 
+    def __init__(
+        self,
+        model_type: str,
+        device: str,
+        backend: str = "",
+        mode: str = "",
+        dynamic=False,
+        fullgraph=False,
+    ):
+        self._model_type = model_type
```

Comment:
_model_type, mmhmm shall we rename it to catagory_name maybe
or group_name?

we do not always benchmark models it can be confusing.

Reply:
Yeah, i'm looking for a better name for this variable. Category_name sounds like a good choice